Plugins/Public/playercntl_plugin/Mail.h

```diff
@@ -13,7 +13,6 @@ namespace Mail
 	void MailShow(const wstring &wscCharname, const string &scExtension, int iFirstMsg);
 	bool MailDel(const wstring &wscCharname, const string &scExtension, int iMsg);
 	void MailCheckLog(const wstring &wscCharname, const string &scExtension);
-	bool MailSend(const wstring &wscCharname, const string &scExtension, const wstring &wscMsg);
```

Comment:
I think this one should stay as-is.

Reply:
I understand why, but I personally think the mail plugin should be part of FLHook Core due to how useful it is as a utility. Perhaps this is something we should consider porting out of Playercntl entirely and moving into Core. 

The ability to send messages while offline is something we'll use again, and I feel I can say that with a degree of certainty. It'll be problematic if we have to integrate a new solution each and every time we do it.